bootstrap.py

```diff
@@ -6,7 +6,7 @@
 
 
 def bootstrap():
-    ConverterRegistry.register(PyGastToCodeConverter, "py")
     ConverterRegistry.register(JsGastToCodeConverter, "js")
+    ConverterRegistry.register(PyGastToCodeConverter, "py")
```

Comment:
What's this about?

Reply:
The order these are placed in the bootstrap file is the order they are taken and looped through in automatic language detection

Reply:
Could we set a priority on the converter? LOW, MEDIUM, HIGH and use that to determine order?

Reply:
or maybe change the call signature to `ConverterRegistry.register(PyGastToCodeConverter, "py", Priority.Low)` and let the converter registry handle this.

Reply:
Just want to avoid the implicit prioritization here that might be confusing to someone later on when adding a new language.